### ISSUE TITLE
need warning about snapshot on source

### ISSUE THREAD

**Issue by nichollri** [NetAppDocs/cloud-manager-backup-restore#19]
Page: [Managing backups for your ONTAP systems](https://docs.netapp.com/us-en/cloud-manager-backup-restore/task-manage-backups-ontap.html)

We need some type of warning on this page about the implications of an ad-hoc backup on the snapshots on the source.
Here is my attempt at wording:

"When creating an ad-hoc backup a snapshot is created on the source volume.  As this snapshot is not part of the normal schedule it will not rotate off.  You may want to manually delete this snapshot from the source volume once the backup is complete.  This will allow blocks related to this snapshot to be freed.  The name of the snapshot will begin with **cbs-snapshot-adhoc**"

**Comment by netapp-tonacki** [comment]
Hey Richard, Thanks for the info.  I will add this information in the docs.  I think the ONTAP [Deleting Snapshot copies topic](https://docs.netapp.com/us-en/ontap/san-admin/delete-all-existing-snapshot-copies-volume-task.html) provides the same information, so I was thinking of just including that link for users to follow.  What do you think?

Thanks, Tom

**Comment by netapp-tonacki** [comment]
Richard, the content is live now, so [take a look](https://docs.netapp.com/us-en/cloud-manager-backup-restore/task-manage-backups-ontap.html#creating-a-manual-volume-backup-at-any-time) and let me know what you think.

Thanks, Tom

**Comment by nichollri** [comment]
Yes I think linking to that on how to delete the snapshot copies is perfect.

**Comment by netapp-tonacki** [comment]
Thanks.
